sample/Fibonacci.bhai

```diff
@@ -0,0 +1,17 @@
+hi bhai
+
+bol bhai "Fibonacci likhenge";
+bhai ye hai start = 0;
+bhai ye hai next = 1;
+bol bhai start;
+bol bhai next;
+bhai ye hai n = 0;
+jab tak bhai (n < 10) {
+  bhai ye hai nextKaNext = start + next;
+  bol bhai nextKaNext;
+  start = next;
+  next = nextKaNext;
+  n += 1;
+  }
+
+bye bhai
```

Comment:
Update your file with this code. Though BhaiLang can work without indentation, but as suggested, you can use it.

```
hi bhai

  bol bhai "Fibonacci likhenge";
  bhai ye hai start = 0;
  bhai ye hai next = 1;
  bol bhai start;
  bol bhai next;
  bhai ye hai n = 0;
  jab tak bhai (n < 10) {
    bhai ye hai nextKaNext = start + next;
    bol bhai nextKaNext;
    start = next;
    next = nextKaNext;
    n += 1;
  }

bye bhai
```